### ISSUE TITLE
Change Background Picture

### ISSUE THREAD

**Issue by gaertnermarkus** [nmsprime/nmsprime#569]
Change Background Picture for Costumer Login.
The current Pic ('nmsprime/public/images/main-pic-3.png) is not the best choice for this kind of management System ... ;-)

**Comment by schmto** [comment]
yes. in case it is a good picture, but i agree with you :) The solution should be: a adaptable (upload) picture in Global Config / Customer Control Center Tab. This should be a easy fix, requiring a DB migration and some small Controller / View adoptions. Good example howto upload files is inside Configfiles Model/Controller

**Comment by schmto** [comment]
Btw: this is a good task for new developer or account executives job candidates for practical test
